### PR TITLE
move to monorepo structure with pnpm

### PR DESCRIPTION
**Pull request by saihaj**
will move to GH actions and setup changesets in follow up PR

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@graphprotocol/graph-cli",
+  "version": "0.36.1",
```
Comment (evaporei) [inline packages/cli/package.json:3]: I just released `0.37.0` you'll just have to change it here ;)

File: packages/cli/tests/cli/validation/call-handler-with-tuple/generated/ExampleSubgraph/ExampleContract.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
+// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (evaporei) [inline packages/cli/tests/cli/validation/call-handler-with-tuple/generated/ExampleSubgraph/ExampleContract.ts:1]: Do you know why this file was added?

File: packages/cli/tests/cli/validation/source-without-address-is-valid/generated/schema.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
+// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (evaporei) [inline packages/cli/tests/cli/validation/source-without-address-is-valid/generated/schema.ts:1]: Same as `ExampleContract.ts`, do you know why is this being added?

**Review by evaporei — CHANGES_REQUESTED** [review]
Almost everything looks fine (just need to change the `package.json` version :wink: ), I'm just curious about the two new files.

Also if I got it right, one of the examples is a submodule right? Is it pointing to [example-subgraph**s**](https://github.com/graphprotocol/example-subgraphs) or just [example-subgraph](https://github.com/graphprotocol/example-subgraph) (without the `s`)?

**Comment by evaporei** [comment]
Another thing is, you might have to apply these changes in your `packages/cli` folder :sweat_smile: https://github.com/graphprotocol/graph-cli/pull/991

File: packages/cli/tests/cli/validation/source-without-address-is-valid/generated/schema.ts (hunk 3 above)
Comment (saihaj) [reply packages/cli/tests/cli/validation/source-without-address-is-valid/generated/schema.ts]: I forget to update the `.gitignore` paths

File: examples/basic-event-handlers/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+version: '3'
+services:
+  graph-node:
+    image: graphprotocol/graph-node:latest
```
Comment (evaporei) [inline examples/basic-event-handlers/docker-compose.yml:4]: Nice the CI passed :tada:

**Review by evaporei — APPROVED** [review]
🚀

**Pull request merged**